CK3toEU4/Source/Mappers/CultureMapper/CultureMapper.h

```diff
@@ -7,15 +7,22 @@
 #include <optional>
 #include <string>
 
+namespace CK3
+{
+class Culture;
+}
+
 namespace mappers
 {
 class CultureMapper: commonItems::parser
 {
   public:
-	CultureMapper();
+	CultureMapper() = default;
 	explicit CultureMapper(std::istream& theStream);
+	void initializeMapper();
```

Comment:
Initializer functions aren't very idiomatic. Better to have World store ```std::unique_ptr<mappers::CultureMapper>``` and initialize via the constructor.